Gemfile.lock

```diff
@@ -68,10 +68,17 @@ GEM
     diff-lcs (1.3)
     erubi (1.6.1)
     execjs (2.7.0)
+<<<<<<< a09c395442a040a74c8449f93f026b29a040ed91
```

Comment:
Conflict all over the file

Reply:
Please read about how to do rebasing and solve the conflicts